common/rfb/Congestion.cxx

```diff
@@ -70,7 +70,7 @@ static const unsigned MAXIMUM_WINDOW = 4194304;
 
 // Compare position even when wrapped around
 static inline bool isAfter(unsigned a, unsigned b) {
-  return (int)a - (int)b > 0;
+  return a - b > 0 && a - b <= INT_MAX;
```

Comment:
`a - b > 0` is not that clear as it is mostly always true for unsigned integers. Might I suggest `a != b`?

And I wonder if `UINT_MAX/2` is better as it avoids assuming any specific connection between `int` and `unsigned`?

Reply:
done